### PR TITLE
Add pomodoro

### PR DESCRIPTION
**Pull request by G07cha**
screen shot 2015-10-15 at 1 43 39 pm

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ Some good apps written with Electron.
 - [N1](https://github.com/nylas/N1) - Extensible email client.
 - [google-music-electron](https://github.com/twolfson/google-music-electron) - Unofficial Google music app.
 - [Mancy](https://github.com/princejwesley/Mancy) - JavaScript REPL.
+- [Pomodoro](http://g07cha.github.io/pomodoro/) - Timer based on "Pomodoro Technique".
```
Comment (shinnn) [inline readme.md]: `http://g07cha.github.io/pomodoro/` → `https://g07cha.github.io/pomodoro/`

File: readme.md [hunk 2]
```diff
@@ -63,6 +63,7 @@ Some good apps written with Electron.
 - [N1](https://github.com/nylas/N1) - Extensible email client.
 - [google-music-electron](https://github.com/twolfson/google-music-electron) - Unofficial Google music app.
 - [Mancy](https://github.com/princejwesley/Mancy) - JavaScript REPL.
+- [Pomodoro](https://g07cha.github.io/pomodoro/) - Timer based on "Pomodoro Technique".
```
Comment (sindresorhus) [inline readme.md]: > Timer based on the Pomodoro Technique.

**Comment by G07cha** [comment]
Any additional requirements?

**Comment by sindresorhus** [comment]
There's no notification on new commits, so I didn't realize it was fixed. Looks good now.

**Comment by G07cha** [comment]
@sindresorhus, thanks.

**Pull request closed without merging**